tests/test_data/americastestkitchen.com/americastestkitchen_1.json

```diff
@@ -72,13 +72,36 @@
   ],
   "category": "Main Courses, Casseroles",
   "yields": "8 servings",
-  "description": "Could we adapt and simplify this northern Italian classic for the American kitchen?",
+  "description": "Could we adapt and simplify this northern Italian classic for the American kitchen? When we started thinking about a simple lasagna Bolognese recipe, there was no denying the appeal of no-boil noodles. After several tests, we found that a five-minute soak proved most effective for getting sturdy, al dente noodles that bound together the layers of ragu and béchamel without soaking up all the moisture. Stumbling through multiple rounds of meat sauce testing gave us the idea of combining the ragu and béchamel when both were lukewarm. The resulting sauce was thickened but easy to spread, with enough moisture for cooking the noodles in our simplified lasagna recipe.",
```

Comment:
> Generally these changes are OK with me; the two things I'd like to figure out before merge are:
> [ ... snip ... ]
>  * What to do about the lengthy `description` test data.

Given that we will not be merging #1373, I am continuing to think about this text and trying to figure out whether it's acceptable to me as a code reviewer.

Is it part of the recipe, or a typical description of the recipe, like the other test expectations we have today?  Well, it's a recommendation of sorts - preparation guidance - but it's also a description of what the preparers did before deciding on the actual, presented recipe.

This seems to be the style on ATK -- I compared this to some very simple recipes on their site (on the assumption that those might involve less scope for lengthy experimentation descriptions), and found that those also include similarly-formatted elaborations of the path taken to reach the recipe.

If we could choose an alternative recipe where the description simply is a description of the recipe's properties, I'd feel more comfortable; but I don't know if many/any of those exist.

Reply:
> If we could choose an alternative recipe where the description simply is a description of the recipe's properties, I'd feel more comfortable; but I don't know if many/any of those exist.

I spot checked a dozen or so recipes and this seems to be the standard practice for the site. Additionally we would be unable to provide a backup test for the old format without creating it ourselves as this is no longer the format of the live html.

Reply:
Was just ~~eavesdropping on~~ following along the copyright conversations in the other (closed) PR #1373 too. This might be a naive question but what's the reason for copyright concern? Is it just the lack of citation within the test cases? Or is this more about the test case itself.

Reply:
> @hhursev any suggestions on how we should proceed here?

We'll go ahead and merge this pretty good PR that solves a real issue reported by a user of the package.

@jknndy I'm sorry for the long convo herе, mostly unrelated to the code in the PR. Thanks for taking care of the legacy format too..

---

Our stance on the "copyright issue" (if there is one) is pretty clear and we've included a [test_data/LICENSE](https://github.com/hhursev/recipe-scrapers/blob/ea0f9dd9606e54f8b3d5f621ce01e928ca8badea/tests/test_data/LICENSE.md) on that matter.

Reply:
To not leave the @oahnh's question hanging... I personally have no copyright concerns here.